index.js

```diff
@@ -1,102 +1,90 @@
 var crypto = require('crypto')
-  , util = require('util')
-  , Promise = require('bluebird')
-  , EventEmitter = require('events').EventEmitter
-
-function defaultCondition() {
-  return true
-}
+var Promise = require('bluebird')
 
 module.exports = function (client) {
   return new Rollout(client)
 }
 
 function Rollout(client) {
-  EventEmitter.call(this)
```

Comment:
No desire to keep this an `EventEmitter`?

Reply:
The “ready” event was only useful for testing, but `.then`-ing the `.handler` Promise more-clearly describes the asynchronous nature of the function.

“ready” tells you nothing. It could be for a different handler call. The EventEmitter is unnecessary and confusing.